codecov.yml

```diff
@@ -12,4 +19,12 @@ coverage:
       default:
         target: 100%
 
-comment: off
+# We don't want to receive general PR comments about coverage.
```

Comment:
These are already the defaults, but I have added them here to make it explicit and help undestand how the whole codecov.io  process works